plugins/woocommerce/src/Internal/Admin/WcPayWelcomePage.php

```diff
@@ -146,4 +152,28 @@ private function is_user_in_treatment_mode() {
 		return $abtest->get_variation( self::EXPERIMENT_NAME ) === 'treatment';
 	}
 
+	/**
+	 * Checks if there is another payment gateway installed using PaymentGatewaySuggestions.
+	 *
+	 * @return bool Whether there is another payment gateway installed.
+	 */
+	private function is_another_payment_gateway_installed() {
```

Comment:
Hey @ismaeldcom Can we use `WC()->payment_gateways->payment_gateways()` instead? I think it returns the installed payment gateways.

Reply:
Thanks for the suggestion @chihsuan! It is what I used at first with `WC()->payment_gateways()->get_payment_gateway_ids()`. Unfortunately, there are some gateways that where the ID doesn't match with the `PaymentGatewaySuggestions` ones.

That's why I ended up comparing the plugin slug from `get_plugins()` and the `plugins` prop on `PaymentGatewaySuggestions`.

I've should mention it on the PR description 🙂 

Another solution could be adding a static list of gateways ID to check against the active ones. Although reusing the suggestion sounds simpler.

Reply:
I've finally added a static list in 19941b2 to have better control on which gateways will be required to meet the promo requirements. As for now we want to check for any other US based gateway on WC store.